src/less/section-title/base/section-title.less

```diff
@@ -0,0 +1,225 @@
+/* stylelint-disable no-descending-specificity */
+
+.section-title {
+    display: flex;
+    margin: 30px 0 10px 16px;
+}
+
+.section-title__title {
+    max-width: 75%;
+
+    > h2 {
+        margin: 0;
+    }
+}
+
+.section-title__subtitle {
+    color: @color-grey5;
+}
+
+.section-title__cta {
+    margin: 0 16px;
+    white-space: nowrap;
+
+    > a {
+        border-left: 2px solid @color-grey2;
+        display: inline-block;
+        padding-left: 16px;
+    }
+}
+
+.section-title__cta--no-text {
+    margin: 0;
+
+    > a {
+        border: 0;
+        padding: 0;
+    }
+
+    > a > span:not(.section-title__cta-icon) {
+        display: none;
+    }
+}
+
+.section-title__cta > a,
+.section-title__title > h2 > a {
+    color: @color-black;
+    text-decoration: none;
+
+    &:hover {
+        color: @color-black;
+        text-decoration: underline;
+    }
+
+    &:visited {
+        color: @color-black;
+    }
+}
+
+.section-title__cta-icon {
+    display: inline-block;
+    margin-left: 8px;
+
+    span& {
+        .background-icon-image-mixin('arrow-right-bold', 14px, 14px);
+    }
+
+    svg& {
+        height: 14px;
+        width: 14px;
+    }
+}
+
+.section-title__info {
+    margin: 0 24px 0 8px;
+
+    button.icon-btn.infotip__host {
+        height: 28px;
+        vertical-align: middle;
+        width: 28px;
+    }
+}
+
+.section-title__overflow {
+    margin: 0 24px 0 auto;
+
+    button.icon-btn {
+        height: 28px;
+        width: 28px;
+    }
+}
```

Comment:
These two are very similar, can we combine?
Also I think `vertical-align:middle;` might not do anything, since it works only on table's (and some inline elements). We should use `display:flex` and `align-items:center` which is the flexbox variant and works better.

Reply:
+1 for flexbox.

Reply:
Removed `vertical-align`. It wasn't doing anything.

Also refactored styling to share width/heigh styling. For margin, I kept separate selectors since the margin-left part changes.